Css/reset.css

```diff
@@ -1,6 +1,89 @@
 * {
     margin: 0;
     padding: 0;
-    border: 0;
+}
+
+ul,
+ol {
     list-style: none;
-}
\ No newline at end of file
+}
+
+a  {
+    color: inherit;
+    text-decoration: none;
+}
+
+button {
+    color: inherit;
+    font-size: 10px;
+    background: none;
+    border: none;
+}
+
+select {
+    border: none;
+    appearance: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-weight: 400;
+    font-size: inherit;
+}
+
+img {
+    width: 100%;
+}
+
+/*
+button:
+color: зависимый
+шрифт: зависимыый
+бэкграунд: отсутствует
+отступы внутренние: отсутствует
+*/
+
```

Comment:
better to remove it from this file. If you want to keep it for yourself make a copy in _temp folder.